src/client/styles/components/_profile-banner.scss

```diff
@@ -30,33 +31,35 @@
         background-color: #555555;
         z-index: 1;
         float:left;
+        object-fit: cover;
```

Comment:
can you also add `object-position: center` so that the center will be cropped when it's not 1:1 ratio? but this could already be the default also, in which case then it's not necessary to add